src/screens/ApplicantViewer/ApplicantSearch.jsx

```diff
@@ -120,7 +134,7 @@ class ApplicantSearch extends React.Component {
             </DropdownItem>
           </DropdownMenu>
         </ButtonDropdown>
-        <Button onClick={this.onShowFilterModal}>Filter</Button>
+        <Styled.Button onClick={this.onShowFilterModal} style={{ color: "black" }}>Filter</Styled.Button>
```

Comment:
`style={{ color: "black" }}` We should use a class for black color instead of manually assigning the style to the element. That is cleaner and easier to change if required.